test/visual-diff/visual-tests

```diff
@@ -195,8 +195,9 @@
       "url": "examples/visual-tests/amp-list/amp-list.amp.html",
       "name": "AMP List and Mustache",
       "loading_complete_selectors": [
-        ".list1",
-        ".list2"
+        "#list1 > div[role='list']",
+        "#list2 > div[role='list']",
+        "#list3 > div[role='list']",
```

Comment:
should list4 be here as well?

Reply:
`#list4` is the diffable variant which already has this selector visible on page load, so I wasn't quite sure what to use here. I think the first three should be good enough since they'll all be built/laid out in the same batch.